src/settings.ts

```diff
@@ -7,6 +7,15 @@ export interface BrowserSettings {
 }
 
 export function loadSettingsFile (): BrowserSettings {
+  if (process.env.GPTSCRIPT_BROWSER_SETTINGS_FILE != null) {
```

Comment:
if (process.env.GPTSCRIPT_BROWSER_SETTINGS_FILE) is probably better because it can check for empty string too.

Reply:
ESLint didn't like that for some reason. I just added another explicit check against the empty string